src/libxtp/tools/exciton.cc

```diff
@@ -106,13 +106,15 @@ namespace votca {
 
             CTP_LOG(ctp::logDEBUG, _log) << "Saving data to " << _archive_file << flush;
             _orbitals.Save(_archive_file);
-
+            
             Property _summary = _gwbse_engine.ReportSummary();
-            tools::PropertyIOManipulator iomXML(tools::PropertyIOManipulator::XML, 1, "");
-            CTP_LOG(ctp::logDEBUG, _log) << "Writing output to " << _xml_output << flush;
-            std::ofstream ofout(_xml_output.c_str(), std::ofstream::out);
-            ofout << (_summary.get("output"));
-            ofout.close();
+            if(_summary.exists("output")){  //only do gwbse summary output if we actually did gwbse
```

Comment:
this is a whole different thing. Should not be in same branch.

Reply:
It was causing errors in my testing.
This was the patch I submitted into your branch and then also into my own too. IIRC this should have already ended up in main.

Reply:
see #68